### PR TITLE
Sim swapping

### PR DESCRIPTION
**Pull request by Prakhar-Shankar**
Added a control page with name "prevention_guidelines".
The page describes what sim swap fraud is all about and prevention guidelines.
Kindly suggest me any changes if needed.
Issue number - #103 
Plagiarsim check - https://publuu.com/flip-book/216408/518349/page/1

### CODE REVIEW

File: pages/controls/prevention_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## What is sim swap fraud?
```
Comment (kingthorin) [inline pages/controls/prevention_guidelines.md:1]: Please have a look at other files and add some proper front matter.

File: pages/controls/prevention_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+SIM swap fraud is a sophisticated cybercrime that capitalizes on a vulnerability within two-factor authentication and verification processes. In this nefarious scheme, perpetrators exploit the inherent security weaknesses in how mobile phone carriers handle SIM cards. The process unfolds as follows: The scammers initiate contact with the target's mobile carrier, employing deceptive tactics to manipulate the carrier into activating a SIM card that is in the possession of the fraudsters themselves. Once this illicit activation takes place, the scammers effectively wrest control of the victim's phone number. Subsequently, any calls or messages directed to this compromised number unwittingly reach the scammers' device, rather than the legitimate owner's smartphone. This intricate maneuver, commonly referred to as SIM swap fraud, exposes the victim to significant risks and underscores the imperative need for heightened vigilance against such cyber threats.
+
+## How to prevent sim swap fraud?
```
Comment (kingthorin) [inline pages/controls/prevention_guidelines.md:5]: SIM should be full caps here as an acronym

File: pages/controls/prevention_guidelines.md
Comment (kingthorin) [inline pages/controls/prevention_guidelines.md:1]: I'd suggest making this something related to sim swapping. That'll make life easier for future contributors.

Comment (Prakhar-Shankar) [reply pages/controls/prevention_guidelines.md:1]: Sure I'll look into it.

File: pages/controls/prevention_guidelines.md (hunk 1 above)
Comment (kingthorin) [reply pages/controls/prevention_guidelines.md]: https://github.com/OWASP/www-community#creating-new-content

File: pages/controls/SIM_swapping_prevention_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+
+|                title                |    layout     |     author      | contributors  |     tags   |
+| ----------------------------------- | ------------- | --------------- | ------------- | ---------- |
+| SIM swapping prevention guidelines  | col-sidebar   |                 | kingthorin    |  controls  |
+|                                     |               |                 |               |            |
+
```
Comment (kingthorin) [inline pages/controls/SIM_swapping_prevention_guidelines.md:6]: ---
layout: col-sidebar
title: "SIM Swapping Prevention Guidelines"
author: "Prakhar-Shankar"
contributors: ["kingthorin"]
tags: ["control", "SIM Swapping"]

---

{% include writers.html %}

Comment (kingthorin) [inline pages/controls/SIM_swapping_prevention_guidelines.md:6]:
```suggestion
---
layout: col-sidebar
title: "SIM Swapping Prevention Guidelines"
author: "Prakhar-Shankar"
contributors: ["kingthorin"]
tags: ["control", "SIM Swapping"]

---

{% include writers.html %}

```

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
https://owasp.org/www-community/controls/SIM_swapping_prevention_guidelines

**Pull request merged**